src/mplhep/styles/__init__.py

```diff
@@ -1,11 +1,13 @@
 import sys
+
 from matplotlib.pyplot import style as plt_style
 
+from .alice import ALICE
+from .atlas import ATLAS
+
 # Short cut to all styles

```

Comment:
this is a bit confusing, they can be ordered alphabetically, but why is the comment left stranded in the middle

Reply:
Ah sure, that was my autoformatter in PyCharm. I've moved the comment up.